assets/stylesheets/_components--header-link.scss

```diff
@@ -0,0 +1,32 @@
+li.header-link {
+  float: left;
+  margin-right: 5px;
+  list-style: none;
+
+  a {
+    border: 1px solid transparent;
+    color: color("gray-0");
+    font-size: $font-up-1;
+    font-weight: 600;
+    line-height: $line-height-medium;
+    padding: 10px 17px;
+
+    &:hover {
+      border-radius: 4px;
+      border: 1px solid color("gray-3");
```

Comment:
These also need an "active" style to have the red underline when the user is on a page corresponding to each section. You should be able to copy how the dc-vue-header does this to reuse it here.